<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame{
border:sold 10px rgba(0,0,0);
background-image:url(E:/study/QT_Project/LiveSystem/LiveSystemClient/rsc/background.png);
}

QLineEdit{
color:#8d98a1;
background-color:rgb(255, 255, 255);
font-size:16px;
border-style:outset;
border-radius:10px;
font-style:MingLiU-ExtB;
}
QPushButton{
background:#ced1d8;
border-style:outset;
border-radius:10px;
font-style:MingLiU-ExtB;
}
QPushButton:pressed{
background-color:rgb(224,0,0);
border-style:inset;
font-style:MingLiU-ExtB;
}
QCheckBox{
background:rgba(85,170,255,0);
color:white;
font-style:MingLiU-ExtB;
}
QLabel{
background:rgba(85,170,255,0);
color:white;
font-style:MingLiU-ExtB;
font-size:14px;
}
background-color: rgb(94, 94, 94);</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>2</y>
     <width>400</width>
     <height>300</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QtMaterialAutoComplete" name="lineEdit_name">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>130</y>
      <width>180</width>
      <height>30</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QtMaterialAutoComplete" name="lineEdit_pwd">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>180</y>
      <width>180</width>
      <height>30</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_login">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>230</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>登录</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_user_pic">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>55</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Rsc.qrc">:/rsc/user.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_logo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Rsc.qrc">:/rsc/live.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_name_logo">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>135</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Rsc.qrc">:/rsc/username.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_pwd_logo">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>185</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Rsc.qrc">:/rsc/password.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_font">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>15</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>Live</string>
    </property>
   </widget>
   <widget class="MyToolButtonMin" name="toolButton_min">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>0</y>
      <width>40</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0);</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="MyToolButtonClose" name="toolButton_close">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>0</y>
      <width>40</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/*background-color: rgba(251, 115, 115, 0);*/</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QtMaterialCircularProgress" name="progressBar">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>291</width>
      <height>221</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="MyToolButtonFont" name="toolButton_signin">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0);</string>
    </property>
    <property name="text">
     <string>注册</string>
    </property>
   </widget>
   <widget class="MyToolButtonFont" name="toolButton_getpwd">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>270</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0);</string>
    </property>
    <property name="text">
     <string>找回密码</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyToolButtonClose</class>
   <extends>QToolButton</extends>
   <header location="global">mycontrolwidgets.h</header>
  </customwidget>
  <customwidget>
   <class>QtMaterialAutoComplete</class>
   <extends>QLineEdit</extends>
   <header location="global">qtmaterialautocomplete.h</header>
  </customwidget>
  <customwidget>
   <class>QtMaterialCircularProgress</class>
   <extends>QProgressBar</extends>
   <header location="global">qtmaterialcircularprogress.h</header>
  </customwidget>
  <customwidget>
   <class>MyToolButtonMin</class>
   <extends>QToolButton</extends>
   <header location="global">mycontrolwidgets.h</header>
  </customwidget>
  <customwidget>
   <class>MyToolButtonFont</class>
   <extends>QToolButton</extends>
   <header location="global">mycontrolwidgets.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Rsc.qrc"/>
 </resources>
 <connections/>
</ui>
